channel.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Carlos Alberto (L1ghtzin)",
+			"oem": "Motorola",
+			"device": "Moto G7 Play",
+			"filename": "crDroidAndroid-13.0-20231103-channel-v9.10.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/channel/9.x/crDroidAndroid-13.0-20231103-channel-v9.10.zip/download",
+			"timestamp": 1699041088,
+			"md5": "14395ef21445985e03b6cff3694714f1",
+			"sha256": "f28c7923058cfa0d191b8cd9fbd709447b0b98e137859049b279f6bb1ef9e449",
+			"size": 774873238,
+			"version": "9.10",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-13-0-channel-official-crdroid-v9-10-03-11-2023.4639621/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Releases/NikGapps-T/18-Apr-2023/NikGapps-core-arm64-13-20230418-signed.zip/download",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "",
+			"telegram": "",
+			"dt": "https://github.com/crdroidandroid/android_device_motorola_channel",
+			"common-dt": "",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_motorola_sdm632"
```

Comment:
maybe this one? 
https://github.com/crdroidandroid/android_kernel_motorola_sdm632/tree/13.0-dynamic